### DISCUSSION TITLE
Local Copy of Cloud Project corrupted

### DISCUSSION THREAD

**Discussion by chrysenteron** [opengisch/QField#5132] [Bugs]
Hi
One of the members of my team has had an issue, where the Android app reported, that the local copy of the cloud project had been corrupted and the changes couldn't be pushed to the cloud. Is there a way to resolve this without losing the changes? 
![WhatsApp Image 2024-03-19 at 20 25 10](https://github.com/opengisch/QField/assets/117898590/e53b7c99-498f-4b3e-93c8-f47182bf7f53)

**Comment by BoswachterMarc** [comment]
This is a solution I used before: while the QField cloud project is open on your team member's device, he/she can open the project folder and export the data.gpkg (use the tree dots).
You can load that gpkg file in QGis, save the (desired) local changes and upload to the cloud.

**Reply by chrysenteron** [reply]
Thanks for your reply. In the meantime, we've had a couple more occurences of this problem. Do you know why/how this happens and are there ways to avoid corruption of the project files?  Also, did you get any feedback of others with the same problem, or could this be isolated to our group or this project?

**Reply by suricactus** [reply]
I assume you are using app.qfield.cloud as a service? If you do, can you please the official support form via https://support.qfield.cloud/portal/en/newticket and you will be asked for more details to see what is going wrong with your project?

**Reply by chrysenteron** [reply]
I've done that now, thanks :)

**Reply by Churros396** [reply]
Hey @chrysenteron could you share what caused the error ? I am having a similar issue.
